src/OpenTelemetry.Instrumentation.AspNet/AspNetInstrumentationOptions.cs

```diff
@@ -26,7 +26,7 @@ namespace OpenTelemetry.Instrumentation.AspNet
     public class AspNetInstrumentationOptions
     {
         /// <summary>
-        /// Gets or sets a Filter function to filter instrumentation for requests on a per request basis.
+        /// Gets or sets a Filter function to collect telemetry about requests on a per request basis.
```

Comment:
```suggestion
        /// Gets or sets a Filter function which can decide to collect or not collect telemetry about requests on a per request basis.
```

Reply:
Is this better?

Reply:
`Gets or sets a Filter function that determines whether or not to collect telemetry about requests on a per request basis`?

I'm fine with either. You tell me and I'll update it 👍

Reply:
Gets or sets a Filter function that determines whether or not to collect telemetry about requests on a per request basis -- this looks good for me.

Reply:
Updated.
* AspNet
* AspNetCore
* Http